### ISSUE TITLE
DM tribute Buggy

### ISSUE THREAD

**Issue by Continueur** [elysium-project/classic-bug-tracker#1141]
The mobs run through walls and the boss/ mate run through walls as well. The geodata is glitchy now, was great a few months ago.

**Comment by deartonym** [comment]
Yeah we met with exactly the same problem. And the King Gordok can intercept on the air! that is not acceptable, ridiculous.

**Comment by tranqshot** [comment]
I've gotten it recorded to show the clipped path of Gordok. I think this all happened after last reset?

**Comment by Continueur** [comment]
Yeah I think it’s been going on for a couple weeks. I haven’t been farming DM lately so I’m not sure when it started, it really is annoying though. I hope they fix it soon!

**Comment by Sifern** [comment]
Confirmed to still be an issue! Attempted some DM:T farming today again and Gordok is still running through the wall. This wasn't an issue at all a month ago.

**Comment by eustachozord** [comment]
Hello my friends
I did 100+ solo dm n runs as Hunter.
There are 2 things buged. 
First of all boss and add are going through the wall. Its after last event and never happened befour.
Secound thing is some pathing was changed befour secound dors ( that you have to open with rogue/eng) you cant send pet from the "stairs" where eye is finishing patrol. It stops on wooden thing on left. Its also new thing.

GL on fixing buggs. Will be there any info that it was fixed anywhere ?

**Comment by Sifern** [comment]
Can confirm these are exactly the 2 bugs I faced. Both pathing/map issues. Hopefully they will flag this issue as fixed when it's fixed so everybody who commented here gets a notification on their github e-mail.

**Comment by Ziggy-QA** [comment]
This has been confirmed and forwarded to developers.
